content/IA-update_connections/Overview.textile

```diff
@@ -0,0 +1,293 @@
+
+h3(#getting-started). Getting started: creating a connection
+
+p(caveat box). Connection is implicit with instantiation but this section covers explicit calls to connect and use-cases for when to use this option.
+
+The Ably Realtime library will open and maintain a connection to the Ably realtime servers as soon as it is instantiated. The @Connection@ object provides a straightforward API to monitor and manage connection state.
+
+The following example relies on the default auto-connect behavior of the library, and then subscribes to the connection's @connected@ event.
+
+bc[javascript]. const ably = new Ably.Realtime({ '{{API_KEY}}' });
+ably.connection.on('connected', () => {
+  console.log('Connected to Ably!');
+});
+
+bc[nodejs]. const Ably = require('ably');
+const ably = new Ably.Realtime({ '{{API_KEY}}' });  
+ably.connection.on('connected', () => {
+  console.log('Connected to Ably!');
+});
+
+bc[ruby]. ably = Ably::Realtime.new('{{API_KEY}}')
+ably.connection.on(:connected) do
+  puts "Connected to Ably!"
+end
+
+bc[java]. AblyRealtime ably = new AblyRealtime("{{API_KEY}}");
+ably.connection.on('connected', new ConnectionStateListener() {
+  @Override
+  public void onConnectionStateChanged(ConnectionStateChange change) {
+    System.out.println("Connected to Ably!");
+  }
+});
+
+bc[csharp]. AblyRealtime ably = new AblyRealtime("{{API_KEY}}");
+ably.Connection.On(ConnectionEvent.Connected, args => {
+  Console.WriteLine("Connected to Ably!");
+});
+
+bc[objc]. ARTRealtime *ably = [[ARTRealtime alloc] initWithKey:@"{{API_KEY}}"];
+[ably.connection on:ARTRealtimeConnectionEventConnected callback:^(ARTConnectionStateChange *change) {
+    NSLog(@"Connected to Ably!");
+}];
+
+bc[swift]. let realtime = ARTRealtime(key: "{{API_KEY}}")
+realtime.connection.on(.connected) { change in
+    print("Connected to Ably!")
+}
+
+bc[flutter]. final realtime = ably.Realtime(key: '{{API_KEY}}');
+final channel = realtime
+  .on(ably.ConnectionStateChange.connected)
+  .subscribe((ably.ConnectionStateChange stateChange) {
+    print('Connected to Ably!');
+  }
+);
+
+blang[javascript,nodejs].
+  If using the promises interface:
+
+  bc[javascript]. const ably = new Ably.Realtime.Promise('{{API_KEY}}');
+  await ably.connection.once("connected");
+  console.log('Connected to Ably!');
+
+  bc[nodejs]. const Ably = require('ably');
+  const ably = new Ably.Realtime.Promise('{{API_KEY}}');
+  await ably.connection.once("connected");
+  console.log('Connected to Ably!');
+
+
+<span lang="ruby">Note that all examples on this page assume you are running them within an EventMachine reactor. Find out more in our "Realtime usage documentation":/realtime/usage.</span>
+
+
+h2(#monitoring-connections). Monitoring connections
+
```

Comment:
I think we need some intro text here to explain what we mean by monitoring connections.

Reply:
Done